README.md

```diff
@@ -37,19 +37,21 @@ not the actual file-system path).
 ##### dotfiles
 
 Set how "dotfiles" are treated when encountered. A dotfile is a file
-or directory that begins with a dot ("."). Note this check is done on
-the path itself without checking if the path actually exists on the
-disk. If `root` is specified, only the dotfiles above the root are
-checked (i.e. the root itself can be within a dotfile when when set
-to "deny").
+or directory that begins with a dot (".").
 
+Note: This check is performed on the url pathname.
```

Comment:
Can you define what "url pathname" means here? Currently there is nothing in the documentation that makes reference to what the url pathname would be. Before It just said "path", which was the argument name to the constructor, which was at least continuous. Would it makes sense to keep saying "path" or just define what "url pathname" is and how it relates to the arguments the user specified?